src/DynamoCoreWpf/ViewModels/Core/AnnotationViewModel.cs

```diff
@@ -245,9 +245,11 @@ public bool IsExpanded
             set
             {
                 annotationModel.IsExpanded = value;
```

Comment:
I think a better option would be to listen to the model.IsExpanded OnPropertyChanged...instead of the entire setter
(this can be done in another PR) 

annotationModel.PropertyChanged += OnModelPropertyChanged;
OnModelPropertyChanged(sender, e) {
   if (e.PropertyName == "IsExpanded") {
                      InPorts.Clear();
                OutPorts.Clear();
                if (value)
                {
                    this.ShowGroupContents();       
                }
                else
                {
                    this.SetGroupInputPorts();
                    this.SetGroupOutPorts();
                    this.CollapseGroupContents(true);
                    RaisePropertyChanged(nameof(NodeContentCount));
                }
                WorkspaceViewModel.HasUnsavedChanges = true;
                .............the whole setter content............
   }
}

ANd only keep the getter

Reply:
Ok, let's do that on another PR